### ISSUE TITLE
Surface badges from READMEs

### ISSUE THREAD

**Issue by jamietanna** [ecosyste-ms/repos#490]
Knowing that a user has added the https://unmaintained.tech/ badge would be interesting to see surfaced from the API.

For instance, this could be under `$.metadata.badges`

**Comment by andrew** [comment]
Yeah that would be a cool feature, I did something similar over here last year: https://github.com/ecosyste-ms/ost/blob/main/app/models/project.rb#L1050
